packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1071,6 +1071,21 @@ export class StripeHelper extends StripeHelperBase {
     });
   }
 
+  /**
+   * Issue stripe refund
+   * @param paymentIntentId
```

Comment:
Not sure how relevant it is, or how VS Code uses the JSDoc vs inferred things, but&hellip; is this missing the `@param reason`?

I didn't see any specific errors/warnings after adding the [`valid-jsdoc`](https://eslint.org/docs/latest/rules/valid-jsdoc) rule to the .eslintrc.json file, but I don't think the ESLint is even checking TypeScript files.
Although, it also looks like the `valid-jsdoc` rule was [deprecated](https://eslint.org/blog/2018/11/jsdoc-end-of-life/) circa ESLint v5.10.0 (~2018?).

Reply:
updated